swift-mode-repl.el

```diff
@@ -751,17 +779,103 @@ PROCESS-IDENTIFIER is the process ID."
     (goto-char comint-last-input-end)
     (search-forward expected-output nil t)))
 
+;;;###autoload
+(defun swift-mode:debug-ios-device (project-directory
```

Comment:
This is a matter of taste, the name should be `swift-mode:debug-ios-app-on-device` or something clearer.